### PR TITLE
Add tester size choices to the helm chart

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [X] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Add the changes needed to work with PR NBISweden/Localega-tester#21.
A volume is attached unless the test file size is `small` since it is not certain that the node has enough space to store the large file (40GB).

### CODE REVIEW

File: ega-charts/localega/values.yaml [hunk 1]
```diff
@@ -267,6 +267,7 @@ tester:
   run: false
   name: tester
   log: "debug"
+  size: "small"
   repository: nbisweden/localega-tester
-  imageTag: release.v0.0.1
+  imageTag: latest
```
Comment (jonandernovella) [inline ega-charts/localega/values.yaml:272]: We should always try to use a fixed tag. And `latest` builds should also have their own assignable id.

**Review by jonandernovella — DISMISSED** [review]
Looks good!

File: ega-charts/localega/values.yaml (hunk 1 above)
Comment (jbygdell) [reply ega-charts/localega/values.yaml:272]: Normally yes, this is just a stopgap to be able to run more advanced tests.

**Comment by blankdots** [comment]
@jbygdell Would be nice if the checks would pass so that we can integrate this. Do we know what happens ?

**Comment by jbygdell** [comment]
> @jbygdell Would be nice if the checks would pass so that we can integrate this. Do we know what happens ?

Yepp, the nodes in the travis cluster died.
I tried to set up k3s on travis but didn't get the time to finish it.

**Comment by blankdots** [comment]
that is ok, whenever there is time. we have other issues at this time.

**Review by jonandernovella — APPROVED** [review]

**Review by blankdots — APPROVED** [review]
Approved.

**Pull request merged**